ls1intum/Artemis
# Small Improvements of the Example Submissions Board

### Checklist

### Motivation and Context
This PR introduces some smaller improvements to the Example Submissions Board. It adds access rights check for the delete / creates buttons to solve problems with tutors that are Editors / Instructors at other courses. Additionally the "Yes / No"-String was not translated until now.

### Description
I use the new checkAndSetCourseRights()-method to set the isAtLeastEditor-attribute and then use it on the buttons. I also moved some checks to the top to make the code a bit more readable.

### Steps for Testing

0. You need a user who is a tutor at your course but is at least editor at another course. 
1. Navigate to the Example Submissions Board as a Tutor
> The easiest way to do this is by navigating to the Assessment Dashboard -> Select an Exercise -> Start reviewing an Example Submission -> Use the breadcrumbs to go to the Example Submission Board
2. Check that the Tutor has no Buttons to create or delete Example Submissions
3. Check that toggling the language translates the Yes / No-String shown in the table ("Tutors need to assess this submission")
4. Check that Editors can still create / delete Example Submissions.

### Screenshots
Old Tutor View (unchanged for Editors)
![grafik](https://user-images.githubusercontent.com/26540346/126630540-70cd68f9-fedf-490e-b473-185ea4b46c59.png)

New Tutor view (no Buttons)
![grafik](https://user-images.githubusercontent.com/26540346/126630475-1d9d7d72-85c9-4b65-bfb0-107d0ce55bc0.png)

German translation:
![grafik](https://user-images.githubusercontent.com/26540346/126630619-fbf9d6cf-486f-444b-80bc-1e7cf943432f.png)
